src/configs/index.js

```diff
@@ -64,3 +64,70 @@ export const HOMEPAGE_TITLES = {
 
 export const LOGO = 'HORONDI';
 export const URL_LANGUAGE = 'en';
+
+export const FOOTER_CATALOGS = {
+  0: { title: 'Каталоги' },
+  1: { title: 'Catalogs' }
+};
+export const FOOTER_INFORMATION = {
+  0: {
+    title: 'Інформація',
+    items: [
+      { id: 1, url: '/about-us', item: 'Про нас' },
+      { id: 2, url: '#', item: 'Матеріали' },
+      { id: 3, url: '#', item: 'Оплата і доставка' },
+      { id: 4, url: '#', item: 'Умови конфіденційності' }
+    ]
+  },
+  1: {
+    title: 'Information',
+    items: [
+      { id: 1, url: '/about-us', item: 'About us' },
+      { id: 2, url: '#', item: 'Materials' },
+      { id: 3, url: '#', item: 'Payment & shipping' },
+      { id: 4, url: '#', item: 'Privacy policy' }
+    ]
+  }
+};
+
+export const FOOTER_CONTACTS = {
+  0: {
+    title: 'Контакти',
+    items: [
+      { id: 1, url: '#', item: '+38 012 345 678' },
+      { id: 2, url: '#', item: 'horondi@gmail.com' },
+      { id: 3, url: '#', item: 'Львів, вул.Угорська,2' },
+      {
+        id: 4,
+        url:
+          'https://www.google.com.ua/maps/place/%D0%B2%D1%83%D0%BB%D0%B8%D1%86%D1%8F+%D0%A3%D0%B3%D0%BE%D1%80%D1%81%D1%8C%D0%BA%D0%B0,+2,+%D0%9B%D1%8C%D0%B2%D1%96%D0%B2,+%D0%9B%D1%8C%D0%B2%D1%96%D0%B2%D1%81%D1%8C%D0%BA%D0%B0+%D0%BE%D0%B1%D0%BB%D0%B0%D1%81%D1%82%D1%8C,+79000/@49.8130045,24.0348852,17z/data=!3m1!4b1!4m5!3m4!1s0x473ae7fa9be7c3b5:0xb30b2516d705bae6!8m2!3d49.8130011!4d24.0370739',
+        item: 'Показати на мапі'
+      }
+    ]
+  },
+  1: {
+    title: 'Contacts',
+    items: [
+      { id: 1, url: '#', item: '+38 012 345 678' },
+      { id: 2, url: '#', item: 'horondi@gmail.com' },
+      { id: 3, url: '#', item: 'Lviv, Ugorska str.2' },
```

Comment:
It is just info. Why do you use the URL here?

Reply:
Removed